client/src/components/AddCat/AddCat.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, useCallback } from "react";
+import { useStyles } from './style';
+import { useCats } from "../../contexts/cats.context";
+import { Button } from "../common/components/Button";
+import { useNavigate } from "react-router-dom";
+
+const ADD_CAT_TEXT = 'Add Cat';
+const DESCRIPTION_TEXT = 'Description';
+const IMAGE_TEXT = 'Image';
+const LAST_NAME_TEXT = 'Last Name';
+const FIRST_NAME_TEXT = 'First Name';
+
+export const AddCat = () => {
+    const [firstName, setFirstName] = useState<string>('');
+    const [lastName, setLastName] = useState<string>('');
+    const [image, setImage] = useState<string>('');
+    const [description, setDescription] = useState<string>('');
```

Comment:
You don't have to put the type if it's visible from the value
```suggestion
    const [firstName, setFirstName] = useState('');
    const [lastName, setLastName] = useState('');
    const [image, setImage] = useState('');
    const [description, setDescription] = useState('');
```